### PR TITLE
Removing appId from API Client

### PR DESCRIPTION
**Pull request by bethleha**
@praveenselvam Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,12 @@ Add the following dependency to your pom file
 
 ##Usage :
 
-The client needs to be first instantiated with the appropriate application id and key to be able to use
+The client needs to be first instantiated with the appropriate application key to be able to use
```
Comment (manojlds) [inline README.md]: Not related to this change, but this reads weird - can you change to something like:

> First, the client must be instantiated with the appropriate application key (`appKey`)

**Comment by bbala-github** [comment]
@bethleha this is a breaking change, we need to update the version accordingly.

File: README.md (hunk 1 above)
Comment (bethleha) [inline README.md]: :+1:

**Comment by bethleha** [comment]
@praveenselvam @manojlds @bbala-github @ashwanthkumar @shrechak Code review comments incorporated. Please review and merge this.

**Pull request merged**